.github/workflows/python-validate.yml

```diff
@@ -60,15 +60,15 @@ jobs:
           cd python/packages/sdk
 
           python3 -m pip install black pyproject-fmt
-          python3 -m black --check sdk
+          python3 -m black --check .
```

Comment:
In JS, we have prettier check for whole `node` directory. Could it work that way here as well?

Reply:
Just tested it out and `black` allows you to do that